### DISCUSSION TITLE
How do you change the model to GPT 4?

### DISCUSSION THREAD

**Discussion by javapro13** [Azure-Samples/chat-with-your-data-solution-accelerator#431] [General]
How do you change the model to GPT 4?

**Comment by ross-p-smith** [comment]
How are you deploying this? Through the One Click deploy or are you cloning the repository and using `azd up`. Either way, you will need to change azureOpenAIModel and azureOpenAIModelName

**Comment by adamdougal** [comment]
Heya @javapro13!

As Ross mentioned, you can modify the following parameters to change the version of OpenAI.


https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/bd598ad10630d4abf33cce5003471d64c0e30f78/infra/main.bicep#L78-L84


These can either be set via the UI if using one click deploy, or via `azd env set AZURE_OPENAI_MODEL gpt-4` and  `azd env set AZURE_OPENAI_MODEL_NAME gpt-4` if using `azd`.
